src/test/regress/expected/alter_distributed_table.out

```diff
@@ -837,5 +837,41 @@ SELECT * FROM mat_view ORDER BY a;
  2 | 2
 (2 rows)
 
+CREATE TABLE distributed_table_1 (
+  col_1 int,
+  "col\'_2" text,
+  col_3 text generated always as (UPPER("col\'_2")) stored
```

Comment:
Lets make sure that generated by default .. sequence works fine as well

Reply:
To clarify, there are three options around generated columns:
1) generated by default .. as identity
2) generated always .. as identity
3) generated always .. as stored

This pr fixes 3), 1) already works fine on master, 2) is still broken.
For 2), I noted below in pr description:
>Also realized that ReplaceTable cannot handle GENERATED IDENTITY
columns and fix for that seems to be as simple as adding OVERRIDING
SYSTEM VALUE to INSERT... SELECT query that we execute in ReplaceTable.
Maybe we should fix this in another pr. #4624

I guess here `generated by default .. sequence` correponds to 1) and in this pr I will add some tests for 1) too.

Reply:
ah, seems that 1) depends on #4625

Reply:
Plus, even if #4625 is merged, we still don't know how to create new table in `ReplaceTable` since we don't have the logic to deparse such expresssions 1) & 2) in deparser :/